BlitzBoard/Features/LoginScreen.storyboard

```diff
@@ -147,7 +148,7 @@
         </scene>
     </scenes>
     <resources>
-        <image name="LoginScreenIcon" width="564" height="564"/>
+        <image name="LaunchScreen" width="564" height="564"/>
```

Comment:
Is this correct that you want Login Screen to have Launch screen image? If so, the naming is just confusing then :)

Reply:
okay ill change it